scripts/data_api_service

```diff
@@ -2,16 +2,26 @@
 
 script_dir=$(dirname "$(readlink -f "$0")")
 
+logfile=$script_dir/log
+
+rm -f $logfile
+
 if [ -z $KB_DEPLOYMENT_CONFIG ]
 then
     export KB_DEPLOYMENT_CONFIG=$KB_TOP/deployment.cfg
 fi
 
+# something in python import complains when $HOME is not set
```

Comment:
A quick comment on this: check_mk apparently does not set $HOME when being run via remote request.  The test scripts shouldn't need a home directory, so setting it to /dev/null if it doesn't already exist should be safe.